osu.Game.Rulesets.Mania/Mods/ManiaModNoHolds.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mania.Objects;
+using osu.Game.Rulesets.Mods;
+using osu.Framework.Graphics.Sprites;
+using System;
+using System.Collections.Generic;
+using osu.Game.Audio;
+using osu.Game.Rulesets.Mania.Beatmaps;
+using osu.Framework.Utils;
+using osu.Game.Overlays.Settings;
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
+
+namespace osu.Game.Rulesets.Mania.Mods
+{
+    public class ManiaModNoHolds : Mod, IApplicableAfterBeatmapConversion
```

Comment:
This one's the old version of this mod right? This file can be removed. Also, `ManiaRuleset` doesn't use the other mod.

Reply:
No, this is the new one, since the old "No Long Notes" name was too long. I did delete the `ManiaModNoLongNotes.cs`, locally before committing, though, so I don't understand why it still appears after pushing.

Reply:
Following discussion in #difficulty-mania, let's go with the name "Hold Off".

Reply:
This is a wonderful name!